### PR TITLE
feat: export data stores using the CLI

### PR DESCRIPTION
**Pull request by mathnogueira**
This PR allows users to export a data store using the CLI

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [x] updated the docs
- [ ] added a test

### CODE REVIEW

File: docs/docs/cli/creating-data-stores.md [hunk 1]
```diff
@@ -68,8 +68,14 @@ spec:
 To apply the configuration, you need a [configured CLI](./configuring-your-cli.md) pointed to the instance you want to apply the data store. Then you just have to enter:
 
 ```
-tracetest data-store apply -f my/data-store/file/location.yaml
+tracetest datastore apply -f my/data-store/file/location.yaml
```
Comment (danielbdias) [inline docs/docs/cli/creating-data-stores.md:71]: Do we need to update this command also on [our version release blog post](https://tracetest.io/blog/v09-release-notes)?

**Review by danielbdias — APPROVED** [review]
Code ok! One thing that we can think about later is how we can add a test to this CLI command on our CI pipeline.

File: docs/docs/cli/creating-data-stores.md (hunk 1 above)
Comment (mathnogueira) [reply docs/docs/cli/creating-data-stores.md:71]: Yes. @kdhamric can you do it, please?

Comment (kdhamric) [reply docs/docs/cli/creating-data-stores.md:71]: Will need @adnanrahic to correct it...

**Pull request merged**